examples/hosts/app-integration/external-data/src/view/appView.tsx

```diff
@@ -24,8 +24,21 @@ export interface IAppViewProps {
 export const AppView: React.FC<IAppViewProps> = (props: IAppViewProps) => {
 	const { model } = props;
 	const taskList = model.baseDocument.getTaskList("task-list-1");
+
+	const clientID = model.getClientID();
+	const [leaderID, setLeaderID] = useState(model.baseDocument.getLeader());
```

Comment:
As written, if the `model` property changes, this state will not get updated. This might not be a problem, but figured I'd call it out. If the `model` object can change, we'd want to introduce a `useEffect` block to update the state accordingly.